warehouse/templates/index.html

```diff
@@ -15,16 +15,29 @@
 
 {% block title %}PyPI - the Python Package Index{% endblock %}
 
-{% macro project_snippet(release) -%}
-<div class="package-snippet">
-  <h3 class="package-snippet__title">
-    <a href="{{ request.route_path('packaging.project', name=release.project.normalized_name) }}">{{ release.project.name }}</a>
-    <span class="package-snippet__version">{{ release.version }}</span>
-  </h3>
-  <p class="package-snippet__description">{{ release.summary }}</p>
+{% set trending_projects_visible = namespace(total=20) %}
+{% set latest_releases_visible = namespace(total=20) %}
```

Comment:
do you need the `namespace`? seems like it could simply be `latest_releases_visible_total = 20`

Reply:
didn't need it at all, logic can be done in Stimulus